### DISCUSSION TITLE
How to invalidate sessions on different domains during logout?

### DISCUSSION THREAD

**Discussion by mkopeyka** [Kong/kong#11168] [Help]
We have this configuration
 - kong
 - keycloak
 - `appA` on `domainA` with kong OpenID Connect plugin
 - `appB` on `domainB` with kong OpenID Connect plugin
 - both applications use the same kong and keycloak
 
The `appA` may redirect to `appB` and `appB` may redirect to `appA`.
Everything works fine except logout. When I do logout in one app another app stay authenticated.
Is it possible to invalidate both sessions during logout?

**Comment by hbagdi** [comment]
It seems you are using Kong Enterprise, please reach out to our support team at support@konghq.com and they will be able to help you out quickly.
